week2/2-website/css/style.css

```diff
@@ -1,19 +1,261 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+*{
 
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+}
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    height: 100vh;
+}
+
+
+.devices{
```

Comment:
Good use of flex, although it's a little disorganised to have the devices section at the top of the css document.